### DISCUSSION TITLE
View (downloaded) pdf temporarily and delete after viewing?

### DISCUSSION THREAD

**Discussion by fritzrehde** [qutebrowser/qutebrowser#7463] [Q&A] [answered]
I want to improve my pdf (pre)viewing in qutebrowser. Is it possible to download a pdf given its link, open it (with zathura) and then delete the downloaded file once zathura exits. I am guessing this can be achieved by a shell script, I am just unsure how such a shell script would integrate with qutebrowser. Thanks!

**Comment by The-Compiler** [comment, marked as answer]
Press the keybindings shown in the download dialog:

![image](https://user-images.githubusercontent.com/625793/200010188-bf8da034-d603-45d7-be6b-2d5a58ef00ec.png)
